CONTRIBUTING.md

```diff
@@ -690,3 +698,36 @@ When merging pull requests, follow these guidelines:
   It should be valid Markdown, be comprehensive, read like a changelog entry,
   and assume that the reader has no access to the diff.
 * Delete branches for merged PRs (by maintainers pushing to the main repo).
+
+### Marking PRs as "deferred"
+
+We sometimes use the ["deferred" label](https://github.com/python/typeshed/labels/deferred)
+to mark PRs and issues that we'd like to accept, but that are blocked by some
+external factor. Blockers can include:
+
+- An unambiguous bug in a type checker (i.e., a case where the
+  type checker is not implementing [the typing spec](https://typing.readthedocs.io/en/latest/spec/index.html)).
+- A dependency on a typing PEP that is still under consideration.
+- A pending change in a related project, such as stub-uploader.
+
+PRs should only be marked as "deferred" if there is a clear path towards getting
+the blocking issue resolved within a reasonable time frame. If a PR depends on
+a more amorphous change, such as a type system change that has not yet reached
+the PEP stage, it should instead be closed.
+
+Maintainers who add the "deferred" label should state clearly what exactly the
+blocker is, usually with a link to an open issue in another project.
+
+### Closing stale PRs
+
+To keep the number of open PRs manageable, we may close PRs when they have been
+open for too long. Specifically, we close open PRs that either have failures in CI,
+serious merge conflicts or unaddressed feedback, and that have not seen any
+activity in three months.
+
+We want to maintain a welcoming atmosphere for contributors, so use a friendly
+message when closing the PR. Example message:
+
+    Thanks for contributing! I'm closing this PR for now, because it still
+    (fails some tests OR has unresolved review feedback OR has a merge conflict) after three months of inactivity.
+    If you are still interested, please feel free to open a new PR (or ping us to reopen this one).
```

Comment:
Nit: Let's wrap long lines.
```suggestion
    (fails some tests OR has unresolved review feedback OR has a merge conflict)
    after three months of inactivity. If you are still interested, please feel free to open
    a new PR (or ping us to reopen this one).
```

Reply:
> (My last commit only wraps long lines. It doesn't change the wording.)

It does, however, change the line-wrapping by adding forced word-break locations. If this is a block of text that we intend on copy-pasting, may has well save the maintainer from having to fix word wrapping, independent of line-length in the .md file.

Not a blocker, just a nitpick/opinion that this may be a special case.